gatsby-config.js

```diff
@@ -3,8 +3,8 @@ const path = require('path');
 
 module.exports = {
   siteMetadata: {
-    title: `Nextflow Summit 2023`,
-    description: `Nextflow Summit 2023 website`,
+    title: `Nextflow SUMMIT 2023`,
+    description: `Nextflow SUMMIT 2023 website`,
```

Comment:
I was thinking that we should write something more for this description actually, it shows up when sharing a URL..

```suggestion
    description: `Join us at the Nextflow SUMMIT 2023 for the latest developments and innovations from the Nextflow world.`,
```